src/Swarm/Language/Syntax.hs

```diff
@@ -323,8 +324,17 @@ data Const
     AppF
   | -- God-like sensing operations
 
-    -- | Run a command as if you were another robot.
+    -- | Hypothetically run a command as if you were another robot.
     As
+  | -- | Hypothetically run a command as if you were another robot,
```

Comment:
`async` doesn't actually have the sense of "as if you were another robot", does it?